Gemfile

```diff
@@ -48,4 +48,6 @@ else
   # prefer to test against latest version because sorbet is updated frequently
   gem 'sorbet'
   gem 'sorbet-runtime'
-end
\ No newline at end of file
+  gem 'dalli' # Needed by sorbet.
```

Comment:
Does these lines resolve any warning? If yes, maybe we should keep them. I've meant to clean up the warning/unnecessary messages but I haven't done it yet.

Reply:
Let's work them separately.